src/application/applicationPreviewPage.tsx

```diff
@@ -100,6 +104,17 @@ const ApplicationPreviewPage = ({
     }
   };
 
+  const getTargetTitle = (id?: number): string => {
+    if (id) {
+      const target = getSectionFavouriteTarget(id);
+      return `${target?.plot_search_target.lease_identifier || '?'} - ${
+        target?.plot_search_target.lease_address?.address || '?'
+      }, ${target?.plot_search_target.district || '?'}`;
+    } else {
+      return '?';
```

Comment:
Unnecessary else

Reply:
Eh, that's highly subjective. It's not wrong either way in any case, but I find it nicer to put the "otherwise" case of a single branching condition in the else branch since; with multiple ifs I wouldn't decouple the fallback value with the last one, though. Anyway, I left this unchanged.